app/assets/javascripts/embargo_date_field.js

```diff
@@ -0,0 +1,13 @@
+$( document ).ready(function() {
+  $('.datepicker').datepicker({dateFormat:"yy-mm-dd"});
```

Comment:
I think you would want to limit the selector to only the date fields that live in the embargo related portion of the form